auth-saml-ldap/app/root.tsx

```diff
@@ -0,0 +1,53 @@
+import type { MetaFunction } from "@remix-run/node";
+import { useLoaderData } from "@remix-run/react";
+
+import { json } from "@remix-run/node";
+
+
+import { getUser, getSession, sessionStorage } from "./session.server";
+
+import {
+  Links,
+  LiveReload,
+  Meta,
+  Outlet,
+  Scripts,
+  ScrollRestoration,
+} from "@remix-run/react";
+
+export const meta: MetaFunction = () => ({
+  charset: "utf-8",
+  title: "New Remix App",
+  viewport: "width=device-width,initial-scale=1",
+});
+
+export async function loader({ request }: LoaderArgs) {
+  const session = await getSession(request);
+
+  return json({
+    headers: {
+      // only necessary with cookieSessionStorage
+      "Set-Cookie": await sessionStorage.commitSession(session),
+    },
+    user: await getUser(request),
+  });
+}
+
+export default function App() {
+  const { user } = useLoaderData();
+
```

Comment:
This is not used

```suggestion
```

Reply:
Thanks, this particular one I would like to leave for newbies like me to understand that `user` can be accessed in the root for variable navbars etc.

Reply:
as you wish :)